client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -84,6 +86,7 @@ export class ShippingLabelViewWrapper extends Component {
 			// If there are no purchased labels, just show Create labels button
 			if ( ! activeLabels.length ) {
 				return (
+					<span>
```

Comment:
why are these `span`s? Semantically, spans are not allowed to contain block elements. We should use a block element here, and if it needs to be displayed inline, we should adjust its styling.